media.css

```diff
@@ -0,0 +1,1257 @@
+  	width: 100%;
+    height: 100px;
+    background: linear-gradient(2.6deg, #000 64%, #fff 65%);
+    margin-bottom: -5px;
+  }
+
+  .video-section {
+    height: 100%;
+    width: 100%;
+    background-color: #000;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-bottom: 117px;
+    margin-bottom: 0;
+  }
+
+  .video-section__image {
+    display: flex;
+    justify-content: center;
+    order: 1;
+  }
+
+  .video {
+  	display: none;
+  }
+
+  .video-adaptive {
+  	display: block;
+  	width: 88%;
+  	height: 88%;
+  	margin: 0 auto;
+  }
+
+  .video-section__content {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    width: 88%;
+    align-items: center;
+  }
+
+  .video-content__h3 {
+    width: 100%;
+    font-size: 30px;
+    line-height: 37px;
+    margin-bottom: 15px;
+    color: #fff;
+    text-transform: uppercase;
+    text-align: center;
+  }
+
+  .video-content__description {
+    width: 100%;
+    font-size: 20px;
+    line-height: 25px;
+    margin-bottom: 35px;
+    color: #fff;
+    text-align: center;
+  }
+
+  .stories-content {
+	  width: 100%;
+	  padding-bottom: 140px;
+	  background-color: #000;
+	  color: #fff;
+	  margin-top: -5px;
+	  margin-bottom: 0;
+	  padding-bottom: 138px;
+	}
+
+	.stories-content__h4 {
+	  font-size: 30px;
+    line-height: 29px;
+    font-weight: bold;
+	  text-align: center;
+	  text-transform: uppercase;
+	  margin-bottom: 41px;
+	  color: #fff;
+	}
+
+	.stories-items {
+		width: 100%;
+		display: flex;
+		justify-content: space-around;
+	}
+
+	.stories-items__item {
+		width: 40%;
+	}
+
+	.stories-item__image {
+		display: block;
+		width: 100%;
+		
+		margin: 0 auto;
+		margin-bottom: 33px;
+	}
+
+	.stories-item__title {
+		font-size: 25px;
+    line-height: 24px;
+		font-weight: bold;
+		margin-bottom: 9px;
+	}
+
+	.stories-item__description {
+		font-size: 20px;
+    line-height: 25px;
+	  margin-bottom: 39px;
+	}
+
+	.stories-item__description-extended {
+		width: 100%;
+		display: grid;
+	  grid-template-columns: 1.5fr 0.5fr 1fr;
+	}
+
+	.hidden-1024 {
+	  display: flex;
+	  align-items: center;
+  }
+
+	.stories-item__date {
+		font-size: 15px;
+    line-height: 14px;
+	  font-weight: bold;
+	  color: #d1d1d1;
+	  opacity: 1;
+	  text-align: right;
+	}
+
+	.gradient-reverse {
+	  display: block;
+	  position: absolute;
+  	width: 100%;
+    height: 186px;
+    background: linear-gradient(2.6deg, #fff 20%, #000 21%);
+    margin-bottom: -5px;
+    z-index: 0;
+  }
+
+  .gallery {
+  	position: absolute;
+	  width: 100%;
+		padding-bottom: 94px;
+		z-index: 1;
+	}
+
+	.gallery__title {
+		font-size: 18px;
+	  line-height: 17px;
+	  color: #000;
+	  opacity: 0.3;
+	  text-align: right;
+	  text-transform: uppercase;
+	  font-weight: bold;
+	  margin-bottom: 8px;
+	  margin-right: 43px;
+	}
+
+	.gallery-content {
+		display: grid;
+		grid-template-columns: repeat(4, 1fr);
+		grid-template-rows: 2;
+		margin-left: 43px;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/PolinaYartseva_axion/pullRequest?prid=5569682)